src/firebase/auth.js

```diff
@@ -174,6 +174,7 @@ export const signInWithEmail = async (formData, rememberMe) => {
 }
 export const resetUserPassword = async (email) => {
   try {
+    // eslint-disable-next-line no-unused-vars
```

Comment:
just remove the variable definition its useless

Reply:
removed the useless part. forgot password checked, working fine